### PR TITLE
Loading accessibility

### PR DESCRIPTION
**Pull request by mbgower**
Accessibility content update for Loading component.

### CODE REVIEW

**Comment by dakahn** [comment]
@mbgower PRs formatted; should be good to go once it passes

**Review by dakahn — APPROVED** [review]

**Comment by laurenmrice** [comment]
I have added update images to the [boxnote folder](https://ibm.ent.box.com/folder/169409520771). I was not able to make updates to this branch because I need write access.

**Pull request merged**
